Duckling/Numeral/DE/NumParser.hs

```diff
@@ -0,0 +1,159 @@
+{-# LANGUAGE DeriveFunctor #-}
+
+module Duckling.Numeral.DE.NumParser (parseNumeral) where
+
+import Prelude
+import Control.Applicative
+import Data.Char
+import Data.List
+import Data.Foldable
+
```

Comment:
```suggestion
import Data.String

```

Looks like we still have build failures internally - it appears to be related to repo-wide defaults that require Data.String.fromString to be in scope (see https://github.com/google/codeworld/issues/59 which describes the same problem)